### DISCUSSION TITLE
how to switch themes when using css from cdn

### DISCUSSION THREAD

**Discussion by fopsdev** [SAP/fundamental-styles#2007] [Q&A]
hi there

i couldn't find guidelines on how to use the different themes when using the:
```<link href="//unpkg.com/fundamental-styles@latest/dist/fundamental-styles.css" rel="stylesheet">```
approach.

thanks for any enlightment

**Comment by kifflavmraka** [comment]
Having the same problem here, can anyone give a hint? Also, when I download the npm package and load a specific theme, the actual styles are the default ones ( I load sap-fiori3-dark css, nothing dark in the browser.. )
